mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -956,24 +956,29 @@ trait Completions { this: MetalsGlobal =>
         if (matchingType.size == 1) {
           s":${matchingType.head}"
         } else if (matchingType.size > 1) {
-          s"|${matchingType.mkString(",")}|"
+          s"|???,${matchingType.mkString(",")}|"
         } else {
           ":???"
         }
       }
 
       private def fillAllFields(): List[TextEditMember] = {
-        if (allParams.size > 1) {
+        val suffix = "autofill"
+        val shouldShow = allParams.exists(_.toString().startsWith(prefix))
+        val isExpilicitelyCalled = suffix.startsWith(prefix)
+
+        if ((shouldShow || isExpilicitelyCalled) && allParams.size > 1) {
```

Comment:
```suggestion
        if ((shouldShow || isExpilicitelyCalled) && allParams.count(!_.hasDefault) > 1) {
```
For example, `pprint.log` has only one non-default parameter